packaging/linux/common/khiops_env/use_environment_module.sh.in

```diff
@@ -0,0 +1,10 @@
+
+# Try to use environment modules to locate mpiexec if needed
+if ! command -v mpiexec &> /dev/null
+then
+    if source /etc/profile.d/modules.sh &> /dev/null
+    then
+        module unload mpi
+        module load mpi/@MPI_IMPL@-x86_64
```

Comment:
I would try to avoid hard-coding the target architecture here. Perhaps using `CMAKE_SYSTEM_PROCESSOR`?